### DISCUSSION TITLE
is gum choose always expected to return an item?

### DISCUSSION THREAD

**Discussion by pk-actzero** [charmbracelet/gum#322] [Q&A] [answered]
Not sure if this is by design but 
`gum choose "option 1" "option 2" --no-limit` allows me to pick both, one or none. 

When I pick none I still get option 1. My expectation would be that I should get nothing back.
My present use case is more like 
`gum choose "option 1" "option 2" --no-limit --cursor-prefix="[ ] " --selected-prefix="[x] " --unselected-prefix="[ ] " --selected="option 1","option 2"`
The user is prompted to choose at least one but if they decide to pick none I cannot tell the difference. 
Also, the item returned appears to be the last one that was in a selected state.
i.e. in the above if I deselect both and option 1 is the last one deselected I get option 1 back.

**Comment by MikaelFangel** [comment, marked as answer]
This is already solved in the git version as far as I can see (:

**Comment by pk-actzero** [comment]
Tested this with 0.9.0 on ubuntu 22 with the git version, no difference in output. 
Confirmed the same on MacOS

**Comment by MikaelFangel** [comment]
Did you use go install ...@latest to install or which method did you use?

**Reply by pk-actzero** [reply]
ah, just noticed that 0.10.0 is out. Can confirm that resolves it.
